docs/book/src/quick-start.md

```diff
@@ -327,7 +326,7 @@ Volumes:
     DownwardAPI:             true
   azure-identity-token: (Injected by the webhook)
     Type:                    Projected (a volume that contains injected data from multiple sources)
-    TokenExpirationSeconds:  86400
+    TokenExpirationSeconds:  3600
```

Comment:
Default was changed to 1h few releases back
https://github.com/Azure/azure-workload-identity/blob/018b019c014ef00abf07230f08d56892f1d83134/pkg/webhook/consts.go#L29